lib/task_listener.js

```diff
@@ -341,5 +341,7 @@ export default class TaskListener extends EventEmitter {
         });
       }
     }
+
+    this.runtime.stats.record('taskError');
```

Comment:
Is this iike a JS error... In that case I suspect you could add a tag/string `err.code || err.name || 'Error'` it typically has low carnality and is a good way to see what errors plague us the most..